staging/com.scoreSmart/manifest.tt

```diff
@@ -1,26 +1,31 @@
-class @com.scoreSmart {
+class @com.goal_smart {
   import loader from @org.thingpedia.v2();
   import config from @org.thingpedia.config.none();
+  import config from @org.thingpedia.config.none(api_key=$);
+  
+  
+  entity soccer_team #_[description="Soccer Teams in the World"];
+  entity soccer_league #_[description="Soccer Leagues in the World"];
 
   monitorable list query standings(in req league_id: String #_[prompt="what league standings do you want out check out?"],
-                        out Team: String,
-                        out Rank: Number,
-                        out Form: String,
-                        out Points: Number)
-  #_[canonical="get league standings"]
+                        out team: String,
+                        out rank: Number,
+                        out form: String,
+                        out points: Number)
+  #_[result = ["${rank}. ${team} | Form: ${form} | Points: ${points}"]]
+  #_[canonical="the league standings"]
   #_[confirmation="league standings of $league_id"]
-  #_[formatted=[{type="text",text="${Rank}. Team: ${Team} | Pts:  ${Points} | Form: ${Form}"}]]
   #[poll_interval=3600000ms]
   #[doc="Get the standings of a league"];
 
   monitorable list query teamFixtures(in req team_id: String #_[prompt="what team do you want out check out?"],
-                        out League: String,
-                        out Team1: String,
-                        out Team2: String,
-                        out Venue: String)
-  #_[canonical="get team fixtures"]
+                        out league: String,
+                        out team1: String,
+                        out team2: String,
+                        out venue: String)
+  #_[canonical="the team fixtures"]
   #_[confirmation=fixtures of $team_id"]
-  #_[formatted=[{type="text",text="${League} | ${Team1} vs ${Team2} @ ${Venue}"}]]
+  #_[result=["${league} | ${team1} vs ${team2} @ ${venue}"]]
```

Comment:
You might want to write something more conversational, such as:
```
The next game will be ${team1} vs ${team2} at ${venue} for the ${league}.
```
You should aim for something that reads well in voice, rather than something visual.